chromium_src/chrome/browser/ui/cocoa/main_menu_builder.mm

```diff
@@ -37,6 +37,8 @@
 IDS_TAB_CXMENU_CLOSE_DUPLICATE_TABS).command_id(IDC_CLOSE_DUPLICATE_TABS), \
               Item(kCloseOtherTabsResourceId
 
+// TODO(fallaciousreasoning): Add menu item for COMMANDER
```

Comment:
@cdesouza-chromium I don't have a Mac, so I'm a little scared to do this.

Reply:
I'm not sure if I follow what you're trying to say here. I do have a mac if you want me to test something for you though.

Reply:
This is only an issue for Mac? What about Windows/Linux?

Reply:
For Windows/Linux, the menu item is added here:
https://github.com/brave/brave-core/pull/22107/files#diff-2283271e14350311184ccfe17b67de5e026532933dd04faea092caa3611f4871R390

Was a little hesitant to add the menu item back on MacOS, and wasn't 100% sure where it should appear.

Reply:
This seems to be where it was

<img width="369" alt="image" src="https://github.com/brave/brave-core/assets/5474642/b024f8e2-f454-458c-8028-f5d5bd99ab70">

Restored here https://github.com/brave/brave-core/pull/22543